packages/api/cms-api/src/user-permissions/dto/user.ts

```diff
@@ -1,7 +1,8 @@
 import { Field, ObjectType } from "@nestjs/graphql";
+import { CurrentUserInterface } from "src/auth/current-user/current-user";
 
 @ObjectType()
-export class User {
+export class User implements CurrentUserInterface {
```

Comment:
that doesn't seem correct, User implementing CurrentUsterInterface. Shouldn't the interface be completely deleted?